src/applications/mhv-medical-records/reducers/labsAndTests.js

```diff
@@ -100,6 +100,15 @@ const getPractitioner = (record, serviceRequest) => {
   return null;
 };
 
+const getMicrobiologyPractitioner = record => {
+  if (isArrayAndHasItems(record.performer)) {
+    const practitionerRef = record.performer[1]?.reference;
```

Comment:
Microbiology is an interesting case. It can have multiple references in `performer`, for example, one for `Organization` and one for `Practitioner`. However I don't think we can assume that the second one will always be `Practitioner`. We'll need to find out if the ID name can give us a clue. Otherwise we'll have to check to see which is which by doing a lookup for both.

Reply:
You'll have to look into each resource individually to see which is which. See this Slack thread:
https://dsva.slack.com/archives/C04ER7MHX5E/p1719937967138079